airsonic-main/src/main/webapp/WEB-INF/jsp/podcast.jsp

```diff
@@ -1,5 +1,6 @@
-<%@ include file="include.jsp" %>
 <%@ page language="java" contentType="text/xml; charset=utf-8" pageEncoding="iso-8859-1"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
```

Comment:
This change was necessary because this one jsp has a different encoding specified, and the include uses UTF8 because everything else does too.  PMD (rightly) complained that two different encodings were not allowed, so I modified this to include the needed taglib directives directly.

Reply:
What about only using utf-8 ?

Reply:
I don't know why this one had specified a different encoding, but because it was specified (rather than left unset) I have to assume there must have been a reason, so I preserved it.